components/question/NetPromotingScore/index.jsx

```diff
@@ -0,0 +1,66 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+
+import { buildSurveyQuestionsWithAnswers } from '../Details/buildSurveyQuestionsWithAnswers';
+
+export const netPromotingScoreTestIds = {
+  base: 'rating',
+  score: 'rating__score',
+  unlikelyText: 'rating__unlikely-text',
+  likelyText: 'rating__likely-text',
+};
+
+const NetPromotingScore = ({ id, answers, setSurveyQuestionsWithAnswers }) => {
+  const [currentScore, setCurrentScore] = useState(-1);
+  const totalScore = [...Array(answers.length)];
+
+  useEffect(() => {
+    setCurrentScore(-1);
+  }, [id]);
+
+  const OnScoreChange = index => {
+    const surveyQuestionsWithAnswers = buildSurveyQuestionsWithAnswers(id, answers[index].id);
+    setCurrentScore(index);
+    setSurveyQuestionsWithAnswers(surveyQuestionWithAnswer => {
+      surveyQuestionWithAnswer = surveyQuestionWithAnswer.filter(response => response.id !== id);
+      return [...surveyQuestionWithAnswer, surveyQuestionsWithAnswers];
+    });
+  };
+
+  return (
+    <div className="w-[363px] m-auto mt-16" data-test-id={netPromotingScoreTestIds.base}>
+      <ul className="h-14 flex justify-center mb-[15.5px]">
```

Comment:
I just found that there's a `help_text` attribute so maybe add it as `title` and/or `aria-label` on the `ul` ?
![Screen Shot 2565-05-19 at 10 35 22](https://user-images.githubusercontent.com/1772999/169199210-4b3e019c-fb0a-4da3-9b0d-5da4fa043219.png)

Reply:
okay! 😄